src/pages/workspace/taxes/WorkspaceTaxesSettingsCustomTaxName.tsx

```diff
@@ -35,6 +37,17 @@ function WorkspaceTaxesSettingsCustomTaxName({
     const {translate} = useLocalize();
     const {inputCallbackRef} = useAutoFocusInput();
 
+    const validate = useCallback((values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_TAX_CUSTOM_NAME>) => {
+        const errors: FormInputErrors<typeof ONYXKEYS.FORMS.WORKSPACE_TAX_CUSTOM_NAME> = {};
+        const customTaxName = values[INPUT_IDS.NAME];
+
+        if (!ValidationUtils.isRequiredFulfilled(customTaxName)) {
+            errors.name = 'workspace.taxes.errors.customNameRequired';
+        }
+
+        return errors;
+    }, []);
```

Comment:
Hmm, looks like in the Old Dot the limit is 10 characters, @luacmartins should we copy that?

Reply:
I think the limit in OldDot is 8 characters. I'm not sure why that is the case, since the API doesn't seem to limit this, but I think we should make it consistent with OldDot and limit it to 8 characters too.

Reply:
Ok, added!